db/seeds.rb

```diff
@@ -6,11 +6,20 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
+puts "Releasing Pokemons to the wild"
 Pokemon.destroy_all
+temp_pokmeon_user_id = 1
 
-Pokemon.create!(
```

Comment:
Won't work, need to change it

Reply:
Seed file won't work, but won't cause merging issues

Reply:
True, I will make a branch to fix this, thanks